vars/buildDockerImage.groovy

```diff
@@ -3,7 +3,14 @@ void call(Map args = [:]) {
 
     def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
     def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: args.inputManifest))
+    def build_qualifier = inputManifest.build.qualifier
 
+    if (build_qualifier != null && build_qualifier != 'null') {
+        build_qualifier = "-" + build_qualifier
+    }
```

Comment:
Nit: move `else` up.

Reply:
I should just default build_qualifier to null and check inputmanifest.build.qualifier.
Will change in a later improvement.